app/js/components/searchDrug/index.jsx

```diff
@@ -3,10 +3,33 @@ import { debounce } from 'lodash';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
```

Comment:
do we need this space?

Reply:
I will work on this. thanks for the observation